plugins/inventory/scaleway.py

```diff
@@ -105,7 +105,7 @@ def _fetch_information(token, url):
         except Exception as e:
             raise AnsibleError("Error while fetching %s: %s" % (url, to_native(e)))
         try:
-            raw_json = json.loads(response.read())
+            raw_json = json.loads(response.read().decode('UTF-8'))
```

Comment:
Better use:
```suggestion
            raw_json = json.loads(to_text(response.read()))
```
and add `to_text` to the `from ansible.module_utils._text import to_native` in line 92.

Reply:
thx for the code review, I'll commit a fix